src/filtrering/filtrering-filter/filtrering-filter.tsx

```diff
@@ -316,37 +316,66 @@ function FiltreringFilter({filtervalg, endreFiltervalg, enhettiltak, oversiktTyp
                             />
                         )}
                     />
+                    {/* Mellom her kjem det nye hovedmål-filteret */}
+                    <Dropdown
+                        name="Sammenlign gjeldende vedtak og Arena"
+                        id="status-14a-vedtak-filter"
+                        render={() => (
+                            <>
+                                <Alert variant="info" size="small" className="registrering-alert">
+                                    Filteret viser brukere der hovedmål/ innsatsgruppe er ulikt i Arena og det
+                                    iverksatte § 14 a-vedtaket.{' '}
+                                    <Link
+                                        href="https://navno.sharepoint.com/sites/fag-og-ytelser-arbeid-arbeidsrettet-brukeroppfolging/SitePages/Ulike-hovedm%C3%A5l-og-innsatsgruppe-i-Arena,-og-i-iverksatte-%C2%A7-14-a-vedtak(1).aspx"
+                                        target="_blank"
+                                        rel="noreferrer noopener"
+                                    >
+                                        Se mer informasjon på Navet
+                                        <ExternalLinkIcon title="Åpne lenken i ny fane" fontSize="1.2em" />
+                                    </Link>
+                                </Alert>
+                                <CheckboxFilterform
+                                    valg={avvik14aVedtakValg()}
+                                    endreFiltervalg={endreAvvik14aVedtakFilterValg()}
+                                    filtervalg={filtervalg}
+                                    form="avvik14aVedtak"
+                                />
+                            </>
+                        )}
+                    />
+                </div>
+            )}
+            {!visFilter14aFraVedtaksstotte && (
+                <div className="filtrering-filter__kolonne">
+                    <Label size="small">Utfasing av Arena</Label>
```

Comment:
Jeg tenker vi kan bare legge '!visFilter14aFraVedtaksstotte' toggel bare på label, istedenfor på dropdown og slette den nye dropdown med label 'Sammenlign gjeldende vedtak og Arena' slik at vi minimerer kode.

Reply:
Dette var eit godt forslag, eg var også innom tanken på dette sjølv også før eg starta.

Grunnane til at eg ikkje gjorde det var:
- Vi skal ha ulik tekst på dropdown før og etter lansering (for både "name" og "valg")
- Dropdown-en skal liggje inni ulike bolker (div-ar) om toggle er på eller av. Før lansering skal den ha sin eigen, etter skal den vere del av "Gjeldende vedtak § 14 a". Så vidt eg veit kan ikkje dette løysast utan noko duplisering av innerkomponenten.

Sidan det er såpass mange endringar trur eg det er lettare å lese koden med litt duplisering på toppnivå enn om vi flytter logikken inn i komponenten.